src/pkg/logger/logger_test.go

```diff
@@ -83,3 +83,41 @@ func (suite *LoggerUnitSuite) TestPreloadLoggingFlags() {
 	assert.Equal(t, logger.LFText, settings.Format, "settings.Format")
 	assert.Equal(t, logger.PIIHash, settings.PIIHandling, "settings.PIIHandling")
 }
+
+func (suite *LoggerUnitSuite) TestPreloadLoggingFlags_badArgsEnsureDefault() {
```

Comment:
thanks for the test! 😄

Reply:
You're welcome, thank you for the nudge!